2019-07-03-infrastructure-operations-meeting.md

```diff
@@ -0,0 +1,88 @@
+# 2019-07-03 Toronto Workers Co-op: All-Hands Meeting
+
+<sup>[from template][template]</sup>
+
+[:date: Calendar][calendar] | [:cat: GitHub][gh] | [:open_file_folder: GDrive][gdrive] | [:notebook: Meeting Index][meetings]
+
+Time: Wed, June 12, 2019 @ 17.00-18.00 ET    
+Location: https://meet.jit.si/hyphacoop  
+Attending: 
+- elon
+- dc
+- yurko
+- patcon (late)
+
+Notetaker:
+- dc
+
+[TOC]
+
+## Agenda
+
+- Previous Action Items
+    - [ ] dc: record voicemail for Hypha
+    - [ ] patcon: non-urgent task: to switch to using link shortner to version controlled csv in github over gdocs
+    - [x] patcon: land the PR using the links!
+    - [ ] patcon/dc: add link shortner to the handbook
+    - [ ] elon: shepard task tracker selection
+    - [x] dc: fill infra call inbetween all hands
+- Discussion
+    - Task Tracker 
+    - Values and our tools
+        - What values inform our infrastructure decisions?
+        - What do we plan to self-host?
+    - Infra Priorities:
+        - accounting/billing software
+        - CRM
+        - alternative communication method
+
+## Notes
+
+- Checked-in on action items
+- Task Tracker
+    - Loomio polls 2 days for items
+        - all "infra folks" have responded to two questions
+        - internal: GitHub projects still winning
+    - dc: is that poll the final dec'n? I wasn't sure if we needed more documenting based on previous call
+    - patcon: Feeling blocked, want to start using sometime. yurko feels this pain
+    - dc: just wanted to make sure I understood this process
+- Next steps:
+    - Poll is going to close
+    - Poll for next all hands meeting
+- Values and our tools - parked until we had more time
+    - #todo dc to do some set up for that convo later
+- Accounting/billing
+    - Do we need to have accounting software set up RIGHT NOW
+    - More complicated things: auditing, HSTs, payroll
+    - #todo patcon: ask cotech folks and briarwood co-op how they do their books and invoicing
+    - #todo dc: call kenzie; co-op creator; OCA? 
+
+## Follow Up
+
+Next meeting? 
+2 weeks? dc and yurko can't make it
+3 weeks? sounds good -- Jul 24 (All Hands)
```

Comment:
My understanding was that we weren't talking about the all hands slot? So is the #todo this:

```
- #todo patcon: schedule infra call in week of July 24
```

Reply:
```suggestion
3 weeks? yurko away (until jul 26). sounds good -- Jul 24 (All Hands)
```